### PR TITLE
Checkbox group

### PR DESCRIPTION
**Pull request by basz**
implements #416

### CODE REVIEW

File: packages/ember-headless-form/src/-private/components/control/checkbox-group.gts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+    /*
+     * @internal
+     */
+    name: string;
+
+    /*
+     * @internal
+     */
+    selected: string;
```
Comment (simonihmig) [inline packages/ember-headless-form/src/-private/components/control/checkbox-group.gts:23]: Shouldn't this be an array?

```suggestion
    selected: string[];
```

File: packages/ember-headless-form/src/-private/components/control/checkbox-group.gts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+    /*
+     * @internal
+     */
+    setValue: (value: string) => void;
```
Comment (simonihmig) [inline packages/ember-headless-form/src/-private/components/control/checkbox-group.gts:28]: same here

**Review by simonihmig — COMMENTED** [review]
First, apologies for the late review!

This is looking pretty good, thanks so much for working on this!

There seem to be some Glint errors, some of which are probably related to the comments I left below. As soon as this is passing CI, I think we should be good.

@ynotdraw @nicolechung do one or both of you have any spare time to have another look on this?

**Comment by ynotdraw** [comment]
My apologies, I won't have time until early next week to look at this due to other obligations 🙇

**Comment by simonihmig** [comment]
@basz Still some Glint errors showing up in CI. Unfortunately CI does not auto-run as you are a first-time contributor to this repo, so please make sure that `pnpm lint:types` inside of `packages/ember-headless-form` is passing before pushing next time, to not get blocked on me approving the CI runs! :)

**Comment by basz** [comment]
ah.. apologies. passes locally now

**Review by simonihmig — APPROVED** [review]
Looking good! Thanks again @basz!

**Pull request merged**